client/api/eventsApi.js

```diff
@@ -1,12 +1,12 @@
-import { getEncodedToken } from 'authenticare/client'
+import { getAuthorizationHeader } from 'authenticare/client'
 import request from 'superagent'
```

Comment:
had a read through Authenticare docs again, the example used {getAuthorizationHeader }  instead of {getEncodedToken} so changed auth headers to follow example

Reply:
👏🏻 Good catch!